setup.cfg

```diff
@@ -0,0 +1,8 @@
+[flake8]
+max-line-length = 110
+ignore = E133, E226, E228, N802, N803, W503
```

Comment:
What are these and why are they ignored?

Reply:
They are the standard ignore codes documented in the developer guide.

https://developer.lsst.io/coding/python_style_guide.html#exceptions-to-pep-8